python/ray/train/callbacks/results_prepocessors/keys.py

```diff
@@ -16,34 +16,26 @@ class KeysResultsPreprocessor(ResultsPreprocessor):
     def __init__(self,
                  included_keys: Optional[Iterable[str]] = None,
                  excluded_keys: Optional[Iterable[str]] = None) -> None:
-        included_keys = included_keys or []
-        excluded_keys = excluded_keys or []
+        included_keys = included_keys or {}
+        excluded_keys = excluded_keys or {}
 
         keys_to_exclude = [
             key for key in DEFAULT_EXCLUDED_KEYS if key not in included_keys
         ]
         for key in excluded_keys:
             if key in included_keys:
-                logger.error(
+                logger.warning(
```

Comment:
```suggestion
                warnings.warn(
```
also needs an import